app/celer-sim/Transporter.hh

```diff
@@ -45,6 +45,7 @@ struct TransporterInput
     size_type max_steps{};
     bool store_track_counts{};  //!< Store track counts at each step
     bool store_step_times{};  //!< Store time elapsed for each step
+    size_type print_progress{};  //!< Print progress every N events
```

Comment:
Just default this to 1 so that it prints every event, and validate that it's greater than zero

Reply:
Oh, It works with either 0 or 1. So as it is, it already prints all events. Thought that this way would make its use a bit easier...

Reply:
There's extra work that goes into checking zero as a special case. But  maybe it's still fine to do `if (print_progress == 0 || (events % print_progress == 0)) ...` but then you have two different selections mapping to the same behavior, and "print_progress = 0" enabling printing at every event is weird.

Reply:
Agreed. Initialized to `1` and made sure value is positive.